data/oh/legislature/Thomas-E-Brinkman-Jr-3e23faeb-67b5-4f9d-bda3-84be8b1d272c.yml

```diff
@@ -1,28 +0,0 @@
-id: ocd-person/3e23faeb-67b5-4f9d-bda3-84be8b1d272c
```

Comment:
@jessemortenson this person leaves office on 12/31/2022, this looks like their record will be deleted but I think an end date should just be added?